test/extended/util/prometheus/helpers.go

```diff
@@ -45,13 +46,34 @@ type prometheusResponseData struct {
 }
 
 // GetBearerTokenURL makes http request with bearer token
-func GetBearerTokenURL(url, bearer string) (string, error) {
-	cmd := fmt.Sprintf("curl --retry 15 --max-time 2 --retry-delay 1 -s -k -H 'Authorization: Bearer %s' %q", bearer, url)
-	output, err := exec.Command("bash", "-e", "-c", cmd).Output()
+func GetBearerTokenURL(url, bearerToken string) (string, error) {
```

Comment:
/hold
we need to implement a retry strategy

Reply:
retry up to twice in new commit :)

Reply:
hmm the curl command retried up to 15 times? In hindsight, we should probably retry less but with a more generous timeout.

Reply:
retry interval is set to 10 seconds, total timeout is set to 60 seconds in the new commit.